client/main.go

```diff
@@ -3,7 +3,7 @@ package main
 import (
 	"bufio"
 	"encoding/gob"
-	scs "github.com/cpalsulich/simple-chat-server"
+	chat "github.com/cpalsulich/simple-chat-server"
```

Comment:
Totally minor: imports are usually added in two groups: standard library and everything else (automatically).

```
"bufio"
"encoding/gob"
...

chat "github.com/..."
```